sound/soc/sof/sof-priv.h

```diff
@@ -128,6 +129,20 @@ struct snd_sof_platform_stream_params {
 	bool no_ipc_position;
 };
 
+/**
+ * struct snd_sof_module_library_info: 3rd party module library information
+ * @fw: Pointer to the 3rd party module firmware
+ * @fw_offset: Offset of the firmware within the firmware file
+ * @name: 3rd party module firmware name
+ * @id: ID of the module library
+ */
+struct snd_sof_module_library_info {
+	const struct firmware *fw;
+	size_t fw_offset;
+	char *name;
+	u32 id;
```

Comment:
I am so predictable I know: what does this 'id' represent? Who allocates it and why is it needed since we have UUIDs for libraries.

Reply:
It is needed by the firmware and it must be unique number starting from `1`.

Reply:
You missed my point @ujfalusi : We use 'id' everywhere, it's just awful. Can we find a better name that represents the function of that variable?